### ISSUE TITLE
Needs Update. Doesn't Work on Node v19

### ISSUE THREAD

**Issue by Jgordon-pencilwrench** [davidkpiano/frontend-masters-react-workshop#13]
I'm getting numerous errors when attempting to load the application. It seems the only alternative is to run in an older version of node. Disappointing. For a paid course, this repository should be kept up to date I think. My node version:  v19.7.0

Error:

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib/tokenize' 

Fixing that one generates a crypto error. It seems the react version is using md4, a very out of date encryption algorithm.

**Comment by davidkpiano** [comment]
Thanks for pointing this out; will get it fixed soon.

**Comment by Jgordon-pencilwrench** [comment]
Grumpy post aside, I'm enjoying the series. Thanks for putting it together.

**Comment by buenos-nachos** [comment]
Yeah, I've had similar issues. I got a bad engine error when I tried installing via NPM, and then I changed the version number, I got an error with ESLint. Create React App displays a warning message when these errors happen, and I've tried all the basic + involved troubleshooting steps, and nothing seems to work.

To be clear, definitely no hard feelings – it can be hard to keep something you did years ago updated, and I appreciate you leaving a message at all. It's just been an exhausting night, figuring out how to get everything fixed up. I ended up just ripping out the files and pasting them into Vite
